ledger_device_sdk/src/ui/gadgets.rs

```diff
@@ -714,7 +718,7 @@ impl<'a> Field<'a> {
                 .take(chunk_max_lines)
                 .enumerate()
             {
-                chunks[1 + i] = Label::from(core::str::from_utf8(chunk).unwrap_or(""));
+                chunks[1 + i] = Label::from(core::str::from_utf8(chunk).unwrap());
```

Comment:
is assumption about ASCII-only `self.name` and `self.value` holds true, then `core::str::from_utf8(...)` isn't an `Err` for any slice of `self.name` and `self.value`